packages/grafana-traces/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "author": "Grafana Labs",
+  "license": "AGPL-3.0-only",
```

Comment:
Are we sure this is correct? I ask because even though it's the same as core, our other packages use `Apache-2.0` so wondering why there's a difference.

Reply:
Yeah it was my suggestion. Imho if something is for outside consumption (like 3rd party plugins) then it should be `Apache-2.0` but if it's only for our own usage it should stay `AGPL`

Reply:
> Yeah it was my suggestion. Imho if something is for outside consumption (like 3rd party plugins) then it should be `Apache-2.0` but if it's only for our own usage it should stay `AGPL`

Yep, if the code lives in this repository and is not published to NPM as a public package I also believe the `AGPL-3.0-only` licence is correct.

Reply:
Makes sense, thanks for clarifying.